pkg/rke/k8s_rke_system_images.go

```diff
@@ -10646,6 +10646,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled in Rancher v2.8.7
```

Comment:
v2.8.7 is already released, so this would be with Rancher v2.8.8

Reply:
Updated the Rancher version and nginx template according to the template suggested in 2.9 PR